releasenotes/notes/2.0/c-sparse-observable-c14b17a9303b1c34.yaml

```
@@ -1,12 +1,17 @@
 ---
 features:
   - |
-    Introduced a C API to build and interact with sparse observables.
-    While the API surface in this release is fairly small, just covering the 
-    :class:`.SparseObservable` class this new API lays the foundation for Qiskit's C interface. 
-
-    The detailed syntax and more information is available under the C API docs, 
-    but a minimal example to construct the 100-qubit observable ``X0 Y1 Z1`` is:
+    Introduced a new C API to build and interact with sparse observables.
+    While the API surface in this release is fairly small - covering only the 
+    :class:`.SparseObservable` class - it represents an important milestone in the evolution
+    of Qiskit, as this is the first time the SDK exposes a public interface in C. 
+    This is an experimental feature, as Qiskit reserves the right to change the API in a 
+    minor version if necessary, and it lays the foundation for future expansions of
+    the Qiskit C interface. 
```

Small rewording suggestion:

```suggestion
    of Qiskit, as this is the first time the SDK exposes a public interface in C and it lays the
    foundation for future expansions of  the Qiskit C interface. 
    As this is the first public interface in C this is an experimental feature, and if necessary
    the API may change in a minor version release.
```